client/src/App.jsx

```diff
@@ -1,7 +1,26 @@
 import { useState } from "react";
+import AllSpecies from "./components/Species/AllSpecies";
+import IndividualSighting from "./components/Sightings/IndividualSighting";
+import { BrowserRouter, Routes, Route, Link } from "react-router-dom";
+import IndividualSpecies from "./components/Individuals/IndividualSpecies";
 
 function App() {
-  return <></>;
+  return (
+    <BrowserRouter>
+      <div className="title">
+        <Link to="/">Animal Sighting Tracker</Link>
+      </div>
+
+      <Routes>
+        <Route path="/" element={<AllSpecies />}></Route>
+        <Route path="/:id" element={<IndividualSpecies />}></Route>
+        <Route
+          path="/individuals/:id/sighting"
```

Comment:
I thought this might have the same problem you ran into in another project (where it was accidentally picking up the whole path as the `id`), but then I realized that was something in Express, not React.